pyproject.toml

```diff
@@ -25,6 +25,9 @@ dependencies = [
     "tqdm>=4.66.1",
     "wavpack-numcodecs",
     "numpy==2.0.1",
+    "aind-data-schema<2.0",
```

Comment:
Why do you need this (large) dependency that you never use, and cap its version `<2`?